README.md

```diff
@@ -67,9 +114,6 @@ $ pytest test_indexViewModel.py
 $ pytest test_app.py    
```

Comment:
Shouldn't this be `poetry run pytest ...` now?

Also, you need to mention dependencies like having Chrome and chomedriver.exe for the e2e tests.